my-app/src/components/organisms/main/Main.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState} from 'react';
+import Categories from '../../molecules/categories/Categories.js';
+import MenuItems from '../../molecules/menuItems/MenuItems.js';
+import Cart from '../../molecules/cart/Cart.js';
+import catgoriesList from '../../../mocks/categories.json';
+import './styles.css';
+
+export default function Main() {
+  const [categories, setCategories] = useState(catgoriesList);
```

Comment:
Don't use static data. 

Instead emulate API calls and handle asynchrony and loading state as well. Read comments on Ayush's PR to get a better feel of what is to be done here and how you can leverage custom hooks